### PR TITLE
Updating NGCC Flowsheet

### PR DESCRIPTION
**Pull request by JavalVyas2000**
# Updating NGCC Flowsheet

This PR updates and moves the NGCC flowsheet from active/pow_gen to notebooks/doc/flowsheets/pow_gen.

----
Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

    I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
    I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

----
📚 Documentation preview 📚: https://idaes-examples--98.org.readthedocs.build/en/98/

### CODE REVIEW

**Comment by ksbeattie** [comment]
@JavalVyas2000 news?

File: idaes_examples/notebooks/docs/flowsheets/results.txt
Comment (AlexNoring) [inline idaes_examples/notebooks/docs/flowsheets/results.txt:1]: What is this results.txt file that has been added to the HDA and methanol flowsheets? Is this relevant to the NGCC flowsheet?

Comment (JavalVyas2000) [reply idaes_examples/notebooks/docs/flowsheets/results.txt]: No. This was something which I was testing out with. I removed it from this PR.

File: idaes_examples/notebooks/docs/power_gen/ngcc/ngcc.ipynb
Comment (AlexNoring) [inline idaes_examples/notebooks/docs/power_gen/ngcc/ngcc.ipynb:117]: I noticed that none of the cells have been executed in this notebook. All the other example notebooks I've seen have their cells executed, but I'm not sure if this ultimately matters or not.

Comment (AlexNoring) [reply idaes_examples/notebooks/docs/power_gen/ngcc/ngcc.ipynb:117]: @andrewlee94 Should the cells be executed before this is approved?

Comment (andrewlee94) [reply idaes_examples/notebooks/docs/power_gen/ngcc/ngcc.ipynb:117]: That is more a question for @lbianchi-lbl. I do not know exactly how things get run and generated.

**Comment by ksbeattie** [comment]
@JavalVyas2000 any news on this?

File: idaes_examples/notebooks/docs/power_gen/ngcc/ngcc_test.ipynb
Comment (agarciadiego) [inline idaes_examples/notebooks/docs/power_gen/ngcc/ngcc_test.ipynb:33]: Should we think about changing the maintainer?

File: idaes_examples/notebooks/docs/power_gen/ngcc/ngcc_test.ipynb
Comment (AlexNoring) [inline idaes_examples/notebooks/docs/power_gen/ngcc/ngcc_test.ipynb:35]: Please update the date here.

**Review by AlexNoring — APPROVED** [review]
lgtm

**Pull request merged**
